### PR TITLE
liquidity: add force tick endpoint to trigger autoloop in tests

### PR DESCRIPTION
**Pull request by carlaKC**
This PR adds a force tick endpoint for testing purposes, to allow testing the autoloop functionality in itests without needing to wait for ticks (which only trigger every 10 minutes). The endpoint will not work on mainnet, since it is intended for testing only. 

Dependent on #295, only the last 2 commits need review. 

#### Pull Request Checklist
- [x] Update `release_notes.md` if your PR contains major features, breaking changes or bugfixes

### CODE REVIEW

File: cmd/loop/liquidity.go [hunk 1]
```diff
@@ -399,3 +399,24 @@ func suggestSwap(ctx *cli.Context) error {
 
 	return nil
 }
+
+var forceTickCmd = cli.Command{
```
Comment (guggero) [inline cmd/loop/liquidity.go:403]: Do we even need a CLI for this? I assume we'll want to trigger this over RPC in the itests only. If people shouldn't use it, why give them a command?

**Review by guggero — APPROVED** [review]
LGTM, barring the comment about the CLI addition :+1:

**Review by bhandras — APPROVED** [review]
LGTM :ocean:  One comment I have is that we could separate test endpoints in the future to a subserver guarded by build tag.

File: cmd/loop/liquidity.go (hunk 1 above)
Comment (carlaKC) [reply cmd/loop/liquidity.go]: Included it because I used it a bit for testing, but agreed we don't need it. Removed.

**Comment by carlaKC** [comment]
> One comment I have is that we could separate test endpoints in the future to a subserver guarded by build tag.

I think this is actually a really good idea, updated this PR to add a debug server behind a dev tag. Decided against the full subserver setup because it seems unlikely that we'll need multiple servers for loop, debug and swap client should be fine?

File: looprpc/debug.proto [hunk 2]
```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package looprpc;
+
+/**
+DebugClient is a service that exposes endpoints intended for testing purposes.
+These endpoints should not operate on mainnet, and should only be included if
+loop is built with the dev build tag.
+*/
+service DebugClient {
```
Comment (bhandras) [inline looprpc/debug.proto:10]: nit: (subjective): I'd call this simply `Debug`

**Review by bhandras — APPROVED** [review]
Very nice! LGTM

File: loopd/swapclient_debug_server.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// +build dev
```
Comment (guggero) [inline loopd/swapclient_debug_server.go:1]: I think the "idiomatic" naming of the file would be `swapclient_server_debug.go`, the build tag suffix usually comes last. Alternatively we could just add this method to `swapclient_server.go`. It wouldn't be used if not registered. But I guess that comes down to personal preference.

File: looprpc/debug.proto [hunk 4]
```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package looprpc;
+
+/**
```
Comment (guggero) [inline looprpc/debug.proto:5]: nit: double asterisk.

**Review by guggero — APPROVED** [review]
Awesome to have this functionality in a proper way, this should help us test more and more things in the integration tests. Only nits on my part, LGTM :100:

File: loopd/swapclient_debug_server.go (hunk 3 above)
Comment (carlaKC) [reply loopd/swapclient_debug_server.go]: Renamed and moved function + macaroons into `swapclient_server_debug.go`.

**Pull request merged**